.gitignore

```diff
@@ -15,7 +15,7 @@
 .svn/
 
 # D Scanner binaries
-dscanner
+/dscanner
```

Comment:
or `bin/dscanner`

Reply:
Or even `/bin` entirely ? That would also exclude unittest binaries

Reply:
yeah